src/components/header/header.tsx

```diff
@@ -58,16 +101,49 @@ export const HeaderOfApp: React.FC = () => {
   };
   const { t } = useTranslation();
 
+  const itemsWithoutId = [
+    getItem(<Link to="/">{t('Header welcome link')}</Link>, '/'),
+    getItem(<Link to="/login">{t('Header login link')}</Link>, '/login', styles.menu__login),
+    getItem(<Link to="/registration">{t('Header sign up link')}</Link>, '/registration'),
+  ];
```

Comment:
```suggestion
   const itemsWithoutId: ItemType[] = [
    {
      label: <Link to="/">{t('Header welcome link')}</Link>,
      key: '/',
    },
    {
      label: (
        <Link to="/login" key="/login" className={styles.menu__login}>
          {t('Header login link')}
        </Link>
      ),
      key: '/login',
    },

    {
      label: (
        <Link to="/registration" key="/registration">
          {t('Header sign up link')}
        </Link>
      ),
      key: '/registration',
    },
  ];
```
можно так сделать, ну и потом еще даже можно будет упростить, если вынести в массив урлы